content/resources/support/security.md

```diff
@@ -0,0 +1,96 @@
+For Python and Qt libraries, which rely on your operating system, please ensure they are updated regularly using your system's update manager. For Ubuntu/Debian, this typically involves using commands like `sudo apt-get update` and `sudo apt-get upgrade`.
+Other packages are maintained by the community, such as Conda, FlatPak, etc.. Any issue should be raised to the dedicated maintainers.  
+
+#### False positives
+
+Receiving red flags from code scanner alerts can be alarming, but many of the current alerts are false positive, so keep calm and don't panic. 
+
+For example, a common false positive might involve the PostgreSQL client library (libpq) being flagged for server-side vulnerabilities, which do not apply to QGIS's use case.
+
+Read the CVE report, go to the upstream project CVE description and please verify this really concerns. 
+
+Please also communicate these findings to your code scanner vendor for catalog updates, to let a chance this stops in the future. 
+
+There is not much QGIS contributors can do when the classification of dependencies is not accurate in the CVE database.
+
+### Vulnerabilities of QGIS itself
+
+Those are extremely rare but can occur as in any software.
+
+QGIS server is a more critical part than QGIS desktop, since it is exposed to the web on public servers. It has been assessed against SQL injections and various known leaks by big corporations since years now. 
+
+If you believe you have found a security issue, such as vulnerabilities in QGIS or its dependencies, please refer to the [security page]({{< ref "resources/support/security" >}}) for detailed information on how to report it responsibly. Before this, please do the following
+ - Check you are using the latest version of QGIS, and have a glance at the nighlty version for potential ongoing changes (fixes or regressions)
+ - Check if your issue concerns QGIS desktop or QGIS server part
+ - Please check our [Bug tracker](https://github.com/qgis/QGIS/issues) for an already existing and potential fix
+ - then only, raise a private disclosure to the security teams via [our GitHub security page](https://github.com/qgis/QGIS/security). Please avoid publicly disclosing the vulnerability until it has been resolved to prevent potential exploitation.
+
+#### Running python QGIS 
+
+QGIS offers python bindings so that you can extend it by writing plugins, geoprocessing models, actions, project macros, or even rewrite a full application based on pyQGIS. 
+
+In a desktop environment, any scripting language has full access to the user land resources. If you execute blindly external code, it can do a lot of harm. 
+
+QGIS main plugins repository is permissively open, but approval of plugins is still reviewed manually. We do not take advance QA assessment of plugin code. However we forbid to ship compiled code, so you know what you get and are free to audit the code. 
```

Comment:
> However we forbid to ship compiled code, so you know what you get and are free to audit the code.

Is that actually true? For example https://plugins.qgis.org/plugins/qduckdb/version/0.5.0/download/ contains a qduckdb/embedded_external_libs/duckdb/duckdb.cp39-wind_amd64.pyd which is a .dll . There was also a discussion on the ML recently where the Whitebox Geospatial developper considered shipping native code in the plugin